packages/serialize/types/tests.ts

```diff
@@ -16,41 +16,41 @@ const testObjectInterpolation1: ObjectInterpolation<undefined> = {
 }
 
 // $ExpectType SerializedStyles
-serializeStyles({}, [])
+serializeStyles([], {})
```

Comment:
any particular reason why those tests had to be updated?

Reply:
They did not pass on master

Reply:
More specifically: Most of the packages do not pass their own `test:typescript` on `master` and types aren't tested in CI. I don't know about the state of `next` but it's not apparent from CONTRIBUTING.md on which branch development should happen.